### PR TITLE
[FR] localize `Site Reliability Engineering`

### PR DESCRIPTION
**Pull request by guillaumebernard84**
### Describe your changes
localize `Site Reliability Engineering` in French

### Related issue number or link (ex: `resolves #issue-number`)
solves https://github.com/cncf/glossary/issues/2974

### Checklist before opening this PR (put `x` in the checkboxes)
- [x] This PR does not contain plagiarism
  - don’t copy other people’s work unless you are quoting and contributing it to them.
- [x] I have signed off on all commits 
  - [signing off](https://docs.github.com/en/authentication/managing-commit-signature-verification/signing-commits) (ex: `git commit -s`) is to affirm that commits comply [DCO](https://wiki.linuxfoundation.org/dco). If you are working locally, you could add an alias to your `gitconfig` by running `git config --global alias.ci "commit -s"`.

### CODE REVIEW

File: content/fr/site-reliability-engineering.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
+---
+title: Ingénierie de la fiabilité des sites (SRE)
+status: Completed
+category: concept
+tags: ["méthodologie", "", ""]
+---
+
+L'ingénierie de la fiabilité des sites (ou SRE pour Site Reliability Engineering en anglais) est une discipline qui combine l'exploitation informatique et le developpement logiciel.
```
Comment (seb-835) [inline content/fr/site-reliability-engineering.md:8]: développement

**Review by seb-835 — APPROVED** [review]
LGTM 🚀

**Review by sestegra — APPROVED** [review]
LGTM 🚀

File: content/fr/site-reliability-engineering.md [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+---
+
+L'ingénierie de la fiabilité des sites (ou SRE pour Site Reliability Engineering en anglais) est une discipline qui combine l'exploitation informatique et le développement logiciel.
+Ce dernier est utilisé spécifiquement pour résoudre des problèmes d'infrastructure et d'exploitation informatique.
+En d'autres termes, au lieu de créer des fonctionnalités pour une application, les ingénieurs en fiabilité des sites construisent des systèmes pour faire fonctionner les applications.
+Il existe des similitudes avec le [DevOps](/fr/devops/), mais tandis que le DevOps se concentre sur la mise en production du code, le SRE s'assure que le code exécuté en production fonctionne correctement.
+
+## Problème auquel il répond
+
+Garantir le fonctionnement [fiable] (/fr/reliability/) des applications requiert de nombreuses capacités,
```
Comment (Krast76) [inline content/fr/site-reliability-engineering.md:15]: Il y a un espace en trop entre le ] et le (.

**Review by Krast76 — COMMENTED** [review]
Juste une dernière petite chose avant le LGTM :)

File: content/fr/site-reliability-engineering.md (hunk 2 above)
Comment (guillaumebernard84) [reply content/fr/site-reliability-engineering.md]: Bien vu, corrigé

**Review by Krast76 — APPROVED** [review]
LGTM 🚀

**Pull request merged**